README.md

```diff
@@ -14,6 +14,7 @@ Without this data it will not be possible to run OpenRW.
 
  * [Getting Started](https://github.com/rwengine/openrw/wiki/Getting-Started)
  * [FAQ](https://github.com/rwengine/openrw/wiki/FAQ)
+ * [Screenshots & Videos](https://github.com/rwengine/openrw/wiki/Media)
```

Comment:
I'll tidy up the semantics of the section, but I'm hesitant to include images directly in the read-me at this point.